plugins/docs.json

```diff
@@ -0,0 +1,33 @@
+{
+    "Root": {
+        "desc": "A component at the root of any logged-in pages. Can be used for creating modals and pop-ups that display over the whole app. Does not override any existing component.",
+        "props": {}
+    },
+    "PostMessageView": {
+        "desc": "The component displaying the message body of posts for adding new components based on post type. Use in your `postTypeComponents` argument to `registerComponents` in your plugin's intialization function.",
+        "props": {
+            "post": {
+                "type": { "name": "object" },
+                "required": true,
+                "desc": "Post to be displayed"
+            },
+            "mentionKeys": {
+                "type": { "name": "array" },
+                "required": false,
+                "desc": "Array of strings representing words to be interpreted as mentions for the logged-in user"
+            },
+            "compactDisplay": {
+                "type": { "name": "bool" },
+                "required": false,
+                "defaultValue": false,
+                "desc": "Set to true if UI is in compact display mode"
+            },
+            "isRHS": {
+                "type": { "name": "bool" },
+                "required": true,
+                "defaultValue": false,
+                "desc": "Set to true if the post is in the right-hand sidebar (thread, search results, etc.)"
+            }
+        }
+    }
+}
```

Comment:
This file is for documenting any pluggable points that are not overriding any actual components. PostMessageView is a special case since post type overriding happens in a different flow.